framework/console/controllers/BaseMigrateController.php

```diff
@@ -622,6 +622,11 @@ public function actionCreate($name)
         }
 
         list($namespace, $className) = $this->generateClassName($name);
+        // Check that class name less then 251 (255 - strlen('.php'))
+        if (strlen($className) > 251) {
```

Comment:
It should be 180:

https://github.com/yiisoft/yii2/blob/980cfde0f1720a9f8516f76d37afa5a5528882ca/framework/console/controllers/MigrateController.php#L260-L263